bundles/SimpleBackendSearchBundle/src/Controller/SearchController.php

```diff
@@ -141,12 +142,12 @@ public function findAction(Request $request, EventDispatcherInterface $eventDisp
 
             foreach ($params as $paramConditionObject) {
                 //this loop divides filter parameters to localized and unlocalized groups
-                $definitionExists = in_array($paramConditionObject['property'], DataObject\Service::getSystemFields())
-                    || $class->getFieldDefinition($paramConditionObject['property']);
-                if ($definitionExists) { //TODO: for sure, we can add additional condition like getLocalizedFieldDefinition()->getFieldDefiniton(...
+                if (in_array($paramConditionObject['property'], DataObject\Service::getSystemFields())) {
                     $unlocalizedFieldsFilters[] = $paramConditionObject;
-                } else {
+                } elseif ($localizedFields instanceof DataObject\ClassDefinition\Data\Localizedfields && $localizedFields->getFieldDefinition($paramConditionObject['property'])) {
```

Comment:
Could we maybe replace the DataObject\ClassDefinition\Data\Localizedfields whole namespace by an important?
Would make the line shorter and cleaner imho. :)

Reply:
@mattamon Done: https://github.com/pimcore/pimcore/pull/16971/commits/65a80bbea670a7817260b2d746708eba163060ec